cli/tests/testdata/workers/worker_event_handlers.js

```diff
@@ -10,6 +10,7 @@ self.addEventListener("message", (evt) => {
 });
 
 self.onerror = (...args) => {
+  args = args.map((v) => typeof v == "string" ? v.slice(-100) : v);
```

Comment:
Could use a comment explaining why this is done.

Reply:
@nayeemrmn please add a comment so we can land this PR